homeassistant/components/owntracks/config_flow.py

```diff
@@ -12,6 +12,10 @@ def supports_encryption():
     try:
         import nacl  # noqa pylint: disable=unused-import
 
+        # What should I do with that ?
+        # How can it fit #27284 ?
+        # Is it stale code ?
```

Comment:
Need help @frenck @balloob

Reply:
Please do not tag random people. We're all very busy and our notifications are already blowing up. Plus other people can help too. 

solve like this https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/tensorflow/image_processing.py#L10-L13

Reply:
Sorry for the abusive tag